### ISSUE TITLE
Tooltips (e.g. SKOS XL) should be keyboard navigable

### ISSUE THREAD

**Issue by joelit** [NatLibFi/Skosmos#1919]
## Description of the enhancement

We recieved feedback that Skosmos toolitp on the Concept page are not accessible for keyboard users.
Related to #1387 , as it should be implemented first in order to get navigable links as a use case for this issue

## What new functionalities would the enhancement make possible?

 - [X] Tooltips can be activeted with keyboard navigation
 - [ ] Tooltips can be closed with Esc
 - [ ] If the tooltips contain links, they can be focused with keyboard
